doc/source/tracking/paper.md

```diff
@@ -0,0 +1,136 @@
+
+affiliations:
+ - name: University of New Hampshire, USA
+   index: 1
+ - name: The Ohio State University, USA
+   index: 2
+ - name: University of Colorado Boulder, USA
+   index: 3
+ - name: Institution, USA
+   index: 4
+ - name: University of Washington, USA
+   index: 5
+ - name: Institution, USA
+   index: 6
+ - name: Institution, USA
+   index: 7
+ - name: Institution, USA
+   index: 8
+ - name: Institution, USA
+   index: 9
+ - name: University of Toronto, Canada
+   index: 10
+ - name: Institution, USA
+   index: 11
+ - name: Institution, USA
+   index: 12
+ - name: Institution, USA
+   index: 13
+date: 1 September 2022
+bibliography: icepyx_pubs.bib
+
+# Optional fields if submitting to a AAS journal too, see this blog post:
+# https://blog.joss.theoj.org/2018/12/a-new-collaboration-with-aas-publishing
+aas-doi:
+aas-journal:
+---
+
+# Summary
+
+icepyx is both a software library and a community composed of ICESat-2 data users, developers, maintainers, and the scientific community.
+We are working together to develop a shared library of resources - including existing resources, new code, tutorials, and use-cases/examples - that simplify the process of querying, obtaining, analyzing, and manipulating ICESat-2 datasets to enable scientific discovery.
+
+# Statement of need
+
+icepyx aims to provide a clearinghouse for code, functionality to improve interoperability, documentation, examples, and educational resources that tackle disciplinary research questions while minimizing the amount of repeated effort across groups utilizing similar datasets. icepyx also hopes to foster collaboration, open-science practices, and reproducible workflows by integrating and sharing resources.
+
+icepyx began during the cryosphere-themed ICESat-2 Hackweek at the University of Washington in June 2019. At the event, a clear need for a collaborative, shared community space that combined and generalized the tooling (including code and examples past, present, and future) written by Hackweek participants, ICESat-2 Science Team members, and the data user community. This project combined and generalized the existing scripts into a unified framework, adding examples, documentation, and testing where necessary and making them accessible for everyone. The library and community continue to grow and evolve, adding new features and building scientific literacy in open-science, cloud computing, and collaborative development practices. icepyx is now a foundational tool for accessing and working with ICESat-2 data and is featured in multiple scientific publications[@Bisson:2021; @Fernando:2021; @Li:2020], presentations [@js2021agu; @js2020agu; @js2019agu], and Hackweeks [@2022_IS2-HW-tutorials; @2020_IS2-HW-tutorials].
```

Comment:
Need to check if it works, but I'd suggest having each sentence on a separate line so that it's easier for people to make 'suggestion changes' for specific lines rather than the whole paragraph. Similar to what the Acknowledgement section is doing.

```suggestion
icepyx aims to provide a clearinghouse for code, functionality to improve interoperability, documentation, examples, and educational resources that tackle disciplinary research questions while minimizing the amount of repeated effort across groups utilizing similar datasets.
icepyx also hopes to foster collaboration, open-science practices, and reproducible workflows by integrating and sharing resources.

icepyx began during the cryosphere-themed ICESat-2 Hackweek at the University of Washington in June 2019.
At the event, a clear need for a collaborative, shared community space that combined and generalized the tooling (including code and examples past, present, and future) written by Hackweek participants, ICESat-2 Science Team members, and the data user community.
This project combined and generalized the existing scripts into a unified framework, adding examples, documentation, and testing where necessary and making them accessible for everyone.
The library and community continue to grow and evolve, adding new features and building scientific literacy in open-science, cloud computing, and collaborative development practices.
icepyx is now a foundational tool for accessing and working with ICESat-2 data and is featured in multiple scientific publications[@Bisson:2021; @Fernando:2021; @Li:2020], presentations [@js2021agu; @js2020agu; @js2019agu], and Hackweeks [@2022_IS2-HW-tutorials; @2020_IS2-HW-tutorials].
```